src/inverseRobot.js

```diff
@@ -24,7 +24,19 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const robotRevert = {};
+
+  for (const key in robot) {
+    const path = robot[key];
+
+    if (robotRevert[path]) {
```

Comment:
What if `undefined` or `null` are the values of this object?

Find another way how to check if a property exists in an object